src/pages/industries/cloud-providers.jsx

```diff
@@ -0,0 +1,193 @@
+  ],
+  imageSrc: AstronautBee,
+  imageAlt: 'astronaut public speaking bee',
+};
+
+const azureTestimonial = {
+  logo: AzureLogo,
+  description:
+    'Cilium combines with the existing control plane of Azure CNI to bring a high-performance eBPF-based dataplane with extensive security and observability capabilities to Azure Kubernetes Service (AKS)',
+  quotedText:
+    'Azure Kubernetes Service will now be deployed with Cilium open sourced data plane and natively integrated with Azure CNI. Microsoft will handle first-line support and collaborate with Isovalent on specific support issues to their deep knowledge of the technology.',
+  withPerson: true,
+  name: 'Deepak Bansal',
+  role: ' CVP and Technical Fellow, Microsoft Azure',
+  url: 'https://www.youtube.com/watch?v=8it8Hm2F_GM',
+};
+
+const googleTestimonial = {
+  logo: GoogleCloudLogo,
+  description:
+    'Google uses Cilium and eBPF as the new networking dataplane for Google’s managed Kubernetes offering (GKE) and Anthos.',
+  quotedText:
+    'The Cilium community has put in a tremendous amount of effort to bootstrap the Cilium project, which is the most mature eBPF implementation for Kubernetes out there. We at Google actively contribute to the Cilium project, so that the entire Kubernetes community can leverage the advances we are making with eBPF.',
+  withPerson: true,
+  name: 'Gobind Johar',
+  role: 'Product Manager, Google Kubernetes Engine',
+  url: 'https://cilium.io/blog/2020/08/19/google-chooses-cilium-for-gke-networking/',
+  CTAtext: 'Read The Blog Post',
+};
+
+const hertznerTestimonial = {
+  description:
+    'Hetzner Cloud is a German public cloud provider that supplies a variety of cloud products like load balancers, virtual machines, storage, and firewalls',
+  quotedText:
+    "Hetzner was looking to upgrade the company's current ingress architecture to a Kubernetes native solution that a offers ECMP, firewall, IPv6, and DSR capabilities. Overall, Cilium provided a massive increase in both RPS and throughput while still reducing CPU usage.",
+  withPerson: true,
+  name: 'Pol Arroyo',
+  role: 'DevOps Engineer, Hetzner Cloud ',
+  url: 'https://cilium.io/blog/2023/01/25/hetzner-performance-testing/',
+  CTAtext: 'Read The Blog Post',
+  logo: HetznerLogo,
+};
+
+const bulletSection1 = {
+  heading: 'Empowering Azure’s AKS Clusters with Cilium ',
+  withImage: true,
+  paragraphs: [
+    "Cilium's open source data plane is natively integrated with Azure CNI in Azure Kubernetes Service (AKS). This integration allows users to tap into Cilium's feature-rich ecosystem, right from the Azure Marketplace. Microsoft has committed to handling first-line support, ensuring that Azure users benefit from rapid issue resolution and updates.    ",
+  ],
+  imageSrc: CloudImage1,
+  imageAlt: 'cilium big tcp stats',
+};
+
+const bulletSection2 = {
+  heading: "Hetzner Cloud's Adoption of Cilium for a Cloud Native Ingress Architecture",
+  paragraphs: [
```

Comment:
Can we have an image on half the page here?

Reply:
maybe something from the performance testing?

Reply:
how this https://cilium.io/static/f671a92e732bd1a2042cd9f3a22ae006/ad007/architecture.png